recipe/meta.yaml

```diff
@@ -13,33 +13,31 @@ source:
     - patches/make_install.patch
 
 build:
-  number: 4
+  number: 5
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - make
+    # All of this can probably be dropped with GAP 4.11
```

Comment:
@fingolfin the build dependencies do not matter much. They are only installed while we're building this package, so it does not hurt if there are a few tools too many.